ui/app/templates/vault/cluster/clients.hbs

```diff
@@ -10,7 +9,7 @@
 <div class="tabs-container box is-bottomless is-marginless is-fullwidth is-paddingless" data-test-pricing-metrics>
   <nav class="tabs">
     <ul>
-      <LinkTo @route="vault.cluster.clients.index" class="nav-tab-link">
+      <LinkTo @route="vault.cluster.clients.current" class="nav-tab-link">
```

Comment:
Since we deleted the `clients.index` route we need to update the Status dropdown menu link in `cluster-info.hbs`

Reply:
Yep, just got that myself!